backend/build.gradle

```diff
@@ -3,13 +3,10 @@ plugins {
     id 'org.springframework.boot' version '3.2.3'
     id 'io.spring.dependency-management' version '1.1.4'
     id 'org.asciidoctor.jvm.convert' version '3.3.2'
-    id 'java-library'
-    id 'maven-publish'
```

Comment:
@Bryanary-code we can add maven-publish and java-library back in if you need them.